install/migrations/update_10.0.2_to_10.0.3/notifications.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Toolbox\Sanitizer;
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+$itil_types = ['Ticket', 'Change', 'Problem'];
+$iterator = $DB->request([
+    'SELECT' => ['id'],
+    'FROM'   => 'glpi_notifications',
+    'WHERE'  => [
+        'itemtype' => $itil_types,
+        'event'    => 'assign_group',
```

Comment:
Migration should parse all notifications related to ITIL objects, as it should also remove `Notification::ITEM_TECH_IN_CHARGE` and `Notification::ITEM_USER` targets.
```suggestion
```